packages/foreman/foreman/foreman.spec

```diff
@@ -243,7 +243,6 @@ BuildRequires: %{?scl_prefix}rubygem(jwt) < 2.3.0
 BuildRequires: %{?scl_prefix}rubygem(graphql) >= 1.8.0
 BuildRequires: %{?scl_prefix}rubygem(graphql) < 1.9.0
 BuildRequires: %{?scl_prefix}rubygem(graphql-batch)
-BuildRequires: %{?scl_prefix}rubygem-activerecord-nulldb-adapter
```

Comment:
Perhaps move this out of the generated part and add a comment that it's from the nulldb group?